src/remotes.ts

```diff
@@ -3,9 +3,19 @@ import 'dotenv/config';
 const MAPBOX_API_ENDPOINT = 'https://api.mapbox.com';
 const ACCESS_TOKEN = process.env.MAPBOX_API_TOKEN;
 
+type GeocodeApiName = 'mapbox' | 'andrewwebb';
+type GeocodeApiEndpointFn = (remote: Remote) => string;
+type geocodeApis = { [key in GeocodeApiName]: GeocodeApiEndpointFn};
```

Comment:
tiniest nit but missing whitespace

```suggestion
type geocodeApis = { [key in GeocodeApiName]: GeocodeApiEndpointFn };
```